src/firewall/core/ipXtables.py

```diff
@@ -587,7 +587,14 @@ def build_set_policy_rules(self, policy):
             if table == "nat":
                 continue
             for chain in BUILT_IN_CHAINS[table]:
-                rules.append(["-t", table, "-P", chain, _policy])
+                if table == "filter" and policy == "DROP":
+                    p = drop_details[chain]
+                    if p == "REJECT":
+                        # Reject is not implemented.
+                        p = "DROP"
```

Comment:
Why is REJECT not implemented?

Reply:
just a glitch. The patch wrongly moved to the second commit. Fixed now.

The question I have is whether this is the correct way to do it.  E.g. the `if table == "filter" and policy == "DROP":` check.

Reply:
I'm not sure you need to match `table == "filter"`. IIRC the policy can/should be set on the `raw` table as well. Historically, I think the iptables backend blocked _all_ traffic during reload. The nftables only blocks new connections.

I'm okay with changing the iptables behavior to match nftables, but it needs to be a conscious change and probably in a separate commit.

Reply:
so should it be `table in ("filter", "raw")` ? Because there are also tables  `mangle` and `security` (`nat` is handled above).

> I'm okay with changing the iptables behavior to match nftables,

So I don't do that now, right? (I didn't plan to).

Reply:
Correct. Don't do it now.